.gitignore

```diff
@@ -8,3 +8,6 @@ Cargo.lock
 
 # These are backup files generated by rustfmt
 **/*.rs.bk
+
+# emacs temp files
+*~
```

Comment:
```suggestion
*~

```

Reply:
Alternatively editor specific files should typically be ignored with the user's global .gitignore, but does not matter too much.

Reply:
yup thats def one option, seems like good idea, but i'll be keeping this here in case others use similar editors.